.github/workflows/go.yml

```diff
@@ -22,13 +22,13 @@ jobs:
         export PATH=$PATH:$HOME/go/bin
         env
         pwd
-        ls -l /opt/hostedtoolcache/go/1.13.7/x64/bin
+        ls -l $GOROOT/bin
         ls -l $HOME/go/bin
         go version
         curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
         sudo wget -qO- ${OC_BINARY_URL} | sudo tar -xvz -C /bin
         sudo pip install yq
         make test_e2e
       env:
-        OC_BINARY_URL: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.3.0.tar.gz
+        OC_BINARY_URL: http://mirror.openshift.com/pub/openshift-v4/clients/oc/4.3/linux/oc.tar.gz
```

Comment:
Do you know if there's a reason we shouldn't bump the version?

Reply:
i didn't want to bump any other versions in case it broke/needed more changes.